Makefile

```diff
@@ -167,10 +167,11 @@ cfn/lint: | guard/program/cfn-lint
 
 ## Runs eclint against the project
 eclint/lint: | guard/program/eclint guard/program/git
+eclint/lint: TARGET_PROJECT_NAME ?= .
 eclint/lint: ECLINT_PREFIX ?= git ls-files -z | xargs -0
 eclint/lint:
 	@ echo "[$@]: Running eclint..."
-	$(ECLINT_PREFIX) eclint check
+	cd $(TARGET_PROJECT_NAME) && $(ECLINT_PREFIX) eclint check
```

Comment:
Somewhere along the way the `PROJECT_NAME` var is being overridden/dropped. I couldn't figure out where, I even tried defining this var with `?=` in `Makefile.boostrap` (https://github.com/plus3it/tardigrade-ci/blob/master/bootstrap/Makefile.bootstrap#L9) but no dice. I gave up and just made a new variable.

Reply:
Did you try using `export PROJECT_NAME=...`? Maybe _that's_ why the build-harness was doing that 😂

Reply:
I thought I did but realized when looking at it again my edits were being overwritten by the default include.. once I figured that out I was able to test properly and confirm that that is indeed why those export statements are there lol

Reply:
That's why we review! To ask questions in a different way that helps us see the problem a bit differently and find a better solution!